test/functional.js

```diff
@@ -79,7 +79,7 @@ test('loadFromDir', function (t) {
 
 test('generateFromDir', function (t) {
   var js = generateFromDir(testdir)
-  fs.writeFileSync(testmodule, js.replace(/'tacks'/g, "'../../index.js'"))
+  fs.writeFileSync(testmodule, js.replace(/'tacks'/g, "'.." + path.sep + ".." + path.sep + "index.js'"))
```

Comment:
`path.join` would be preferrable to `path.sep`, but `../../` should work fine on Windows.